src/js/jsx/sections/export/ExportList.jsx

```diff
@@ -42,20 +42,6 @@ define(function (require, exports, module) {
         nls = require("js/util/nls");
 
     /**
-     * The options for the scale datalist
-     * @private
-     * @type {Immutable.OrderedMap.<string, {id: string, title: string}>}
-     */
-    var _scaleOptions = Immutable.OrderedMap(ExportAsset.SCALES
```

Comment:
I had to delay this initialization from module-load time to component-mount time because the locale info is not yet ready at module-load time.